int64 num_cameras
string[] filenames
string[] cameraNames
bool onTable

----

int64 objNum            # What object we see
float64[3] trans        # The translation of the object in the world frame
float64[4] quat         # Rotation of the object in the world frame

int64 ret
string msg
